2020/01/index.js

```diff
@@ -1,3 +1,53 @@
+const { promises: fs } = require('fs');
+
+async function getInput() {
+  return input = (await fs.readFile(`${__dirname}/input.txt`))
+    .toString()
+    .trim()
+    .split('\n')
+    .map(i => parseInt(i));
+}
+
+async function p1() {
+  const input = (await getInput())
+    .sort((a, b) => a - b);
+
+  for(let i = 0; i < input.length; i++) {
+    for(let j = i + 1; j < input.length; j++) {
+      const sum = input[i] + input[j];
+      if(sum === 2020) {
+        return input[i] * input[j];
+      } else if(sum > 2020) {
+        break;
+      }
+    }
+  }
+
+  throw new Error('Should not have reached here');
+}
```

Comment:
Definitely not as good as your `Set` solution @faizanminhas :( .

Reply:
haha

Reply:
It's ok, you didn't have to do leet code